pg_search/src/api/aggregate.rs

```diff
@@ -103,6 +103,13 @@ mod pdb {
             _fcinfo: pgrx::pg_sys::FunctionCallInfo,
         ) -> Self::State {
             // This should never execute - if it does, the query wasn't handled by our custom scan
+            if !crate::gucs::enable_aggregate_custom_scan() {
+                pgrx::error!(
+                    "pdb.agg() requires aggregate custom scan to be enabled. \
+                     Set 'paradedb.enable_aggregate_custom_scan = on' to use pdb.agg()."
+                );
+            }
```

Comment:
As mentioned, `enable_aggregate_custom_scan` should not be required if `pdb.agg` is used, since that's an explicit opt-in.

I think that we can consider `enable_aggregate_custom_scan` to be opting into our pushdown of bare/native agg functions like `COUNT`, etc.

Reply:
That's a good idea. Will do it as a follow-up PR.